### PR TITLE
Quickstarts: Improve it/spring-boot-with-jib integration test

### PR DESCRIPTION
**Pull request by mdxabu**
@rohanKanojia, Please review this and merge this.
- [x] Spring Boot version is updated to 2.7.x

- [x] Replace org.springframework.web.bind.annotation.RequestMapping with more specific org.springframework.web.bind.annotation.GetMapping

### CODE REVIEW

**Review by rohanKanojia — APPROVED** [review]

**Review by rohanKanojia — APPROVED** [review]

**Comment by rohanKanojia** [comment]
@mdxabu : Are integration tests failing on your machine ?

**Comment by mdxabu** [comment]
No, Not on my machine

File: it/spring-boot-with-jib/src/main/java/io/fabric8/maven/sample/springboot/jib/HelloController.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package io.fabric8.maven.sample.springboot.jib;
 
-import org.springframework.web.bind.annotation.RequestMapping;
```
Comment (rohanKanojia) [inline it/spring-boot-with-jib/src/main/java/io/fabric8/maven/sample/springboot/jib/HelloController.java:3]: You have removed the import but you haven't replaced the usage with `@GetMapping` (see line 7), that's why you're getting the error.

**Comment by rohanKanojia** [comment]
@mdxabu : Are you sure you tested your changes? After making changes follow these steps:

1. Build a local SNAPSHOT version of docker-maven-plugin by running `mvn clean install -DskipTests` in project base directory
2. Go to `it/spring-boot-with-jib` and run `mvn clean install`

```
[INFO] -------------------------------------------------------------
Error:  COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
Error:  /home/runner/work/docker-maven-plugin/docker-maven-plugin/it/spring-boot-with-jib/src/main/java/io/fabric8/maven/sample/springboot/jib/HelloController.java:[10,6] cannot find symbol
  symbol:   class RequestMapping
  location: class io.fabric8.maven.sample.springboot.jib.HelloController
```

File: it/spring-boot-with-jib/src/main/java/io/fabric8/maven/sample/springboot/jib/HelloController.java (hunk 1 above)
Comment (mdxabu) [reply it/spring-boot-with-jib/src/main/java/io/fabric8/maven/sample/springboot/jib/HelloController.java:3]: @rohanKanojia ,Ok, I will fix this now.

Comment (mdxabu) [reply it/spring-boot-with-jib/src/main/java/io/fabric8/maven/sample/springboot/jib/HelloController.java:3]: @rohanKanojia , On line 10 also, change to `RequestMapping` to `GetMapping` or not?

**Comment by mdxabu** [comment]
@rohanKanojia, It builds successfully
```
[INFO] DOCKER>  D:\Github Projects\docker-maven-plugin\it\spring-boot-with-jib\target\docker\fabric8\spring-boot-dmp-it-jib\tmp\docker-build.tar successfully built
[INFO]
[INFO] --- antrun:1.8:run (generateSources) @ dmp-it-spring-boot-jib ---
[WARNING]  Parameter 'tasks' is deprecated: Use target instead
[WARNING] Parameter tasks is deprecated, use target instead
```

But I got `BUILD FAILURE`
```
main:
[exec] error during connect: Post "http://%2F%2F.%2Fpipe%2FdockerDesktopLinuxEngine/v1.45/images/load?quiet=1": open //./pipe/dockerDesktopLinuxEngine: The system cannot find the file specified.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  10.458 s
[INFO] Finished at: 2024-08-04T16:06:48+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run (generateSources) on project dmp-it-spring-boot-jib: An Ant BuildException has occured: exec returned: 1
[ERROR] around Ant part ...<exec failonerror="true" executable="docker">... @ 4:48 in D:\Github Projects\docker-maven-plugin\it\spring-boot-with-jib\target\antrun\build-main.xml   
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

**Comment by mdxabu** [comment]
@rohanKanojia, Now it `BUILD SUCCESS`

**Comment by mdxabu** [comment]
@rohanKanojia, please review and merge this! By the way, I don't know why the `MacOS E2E tests` are failing on all my commits.

**Pull request merged**